### PR TITLE
Initial Gemini SDK commit

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

File: Examples/GenerativeAISample/ChatSample/Models/ChatMessage.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+enum Participant {
+  case system
```
Comment (ryanwilson) [inline Examples/GenerativeAISample/ChatSample/Models/ChatMessage.swift:18]: Not for this PR but we should update these to `model` cc @peterfriese

File: Examples/GenerativeAISample/GenerativeAISample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,1145 @@
...
+		};
+		8848C83E2B0D04BD007B434F /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_ASSET_PATHS = "\"GenerativeAISample/Preview Content\"";
+				DEVELOPMENT_TEAM = YGAZHQXHH4;
```
Comment (ryanwilson) [inline Examples/GenerativeAISample/GenerativeAISample.xcodeproj/project.pbxproj:781]: Please remove and make an empty string `""`, here and there's one more instance of it

Comment (paulb777) [reply Examples/GenerativeAISample/GenerativeAISample.xcodeproj/project.pbxproj]: Done

File: .github/workflows/cli.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+name: cli
+
+on:
+  pull_request:
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
```
[the remark this answers is not in the document]
Comment (paulb777) [reply .github/workflows/cli.yml:7]: It will let us know if it breaks

**Review by ryanwilson — APPROVED** [review]

**Pull request merged**